bin/ch-test

```diff
@@ -945,6 +967,8 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
+export ch_build_unpack_list
```

Comment:
Subprocesses won't need this; are you sure it needs to be exported?

Reply:
It's my understanding that the `export` is necessary for this variable to be available to other files, e.g. `common.bash`, where we decide whether or not to skip a test. I'm basically copying what we do with `ch_one_test`.